src/task/MonitorTask.ts

```diff
@@ -267,23 +275,17 @@ export default class MonitorTask
       }
     }
 
-    this.reconnectionHealthPolicy.update(connectionHealthData);
-    const reconnectionValue = this.reconnectionHealthPolicy.healthIfChanged();
-    if (reconnectionValue !== null) {
-      this.logger.info(`reconnection health is now: ${reconnectionValue}`);
-      if (reconnectionValue === 0) {
-        this.context.audioVideoController.handleMeetingSessionStatus(
-          new MeetingSessionStatus(MeetingSessionStatusCode.ConnectionHealthReconnect),
-          null
-        );
-      }
-    }
+    this.applyHealthPolicy(this.reconnectionHealthPolicy, connectionHealthData, () => {
```

Comment:
Since you addressed the comments which changed this, could you please check the change log and PR description once  and update it accordingly? For example: `enact` -> `apply